Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -2653,6 +2671,22 @@ - (void)didTapSystemMessageWithInteraction:(TSInteraction *)interaction
     }
 }
 
+- (void)didLongPressSystemMessageCell:(OWSSystemMessageCell *)systemMessageCell;
+{
+    OWSAssert([NSThread isMainThread]);
+    OWSAssert(systemMessageCell);
+    OWSAssert(systemMessageCell.interaction);
+
+    DDLogDebug(@"%@ long pressed system message cell: %@", self.tag, systemMessageCell);
+
+    [systemMessageCell becomeFirstResponder];
+
+    UIMenuController *theMenu = [UIMenuController sharedMenuController];
+
+    [theMenu setTargetRect:systemMessageCell.titleLabel.frame inView:systemMessageCell];
```

Comment:
This assumes that titleLabel's frame is in the coordinate system of systemMessageCell. Probably fine, but we could also convert coordinate systems.